wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -175,11 +176,11 @@ data class WireRun(
    * A map from module dir to module info which dictates how the loaded types are partitioned and
    * generated.
    *
-   * By default a single module is used which includes everything in the root output directory.
+   * When empty everything is generated in the root output directory.
```

Comment:
```
When empty everything
```
 ->
```
When empty, everything
```

Reply:
This is the kind of comment hard to make for non-english-native speakers.